### DISCUSSION TITLE
Resize Avsync

### DISCUSSION THREAD

**Discussion by 7AllenWalker7** [staxrip/staxrip#1030] [General]
Hello, I can't figure out how to resize an image. There is an autosync/vapoursync filter and a size slider. When you select a filter and set the desired size (reduce) without changing the decoder (), the size does not change. When you select a decoder, the size changes (the line --output-res appears), but it is not clear which method of resizing is used, selected for the autosync / vaporsync filter or by default for the decoder.
At the same time, if you select resizing in the decoder, then the autosync / vaporsync filter is still enabled and it is also not clear what is used to resize the image.
![photo](http://ipic.su/img/img7/fs/Snimokekrana2023-06-28125610.1687921003.png)
![photo](http://ipic.su/img/img7/fs/Snimokekrana2023-06-28125546.1687921126.png)

**Comment by Dendraspis** [comment]
When moved, the slider adds the **Resize** filter section and within that it adds a resize filter function. So as long as the filters are used (which always should be the case) the encoded video should be resized.

If you have a reproducible example, where it does not work, please create an issue providing information to reproduce it, best with a short sample file.

Resize options via encoders are of course different and not adopted by StaxRips UI. In case of using them you have to be careful to not mix their settings or use double-resizes.

**Comment by 7AllenWalker7** [comment]
`Resize options via encoders are of course different and not adopted by StaxRips UI. In case of using them you have to be careful to not mix their settings or use double-resizes.`
when I select resize in the encoder, the slider changes the parameters, but the filter is also enabled. It's confusing.

`If you have a reproducible example, where it does not work, please create an issue providing information to reproduce it, best with a short sample file.`
Filter resizing does not work if no encoder is selected (set Copy/Mux). You can't just change the size without changing the encoding settings of the files.

**Reply by Dendraspis** [reply]
Which encoder and encoder setting do you mean?

Resizing always means you have to re-encode the entire video - there is no way to resize the video while keeping the quality.

**Reply by 7AllenWalker7** [reply]
okay,  now I understood)
